src/builtins/core/duration.rs

```diff
@@ -602,204 +602,239 @@ impl Duration {
         self.add(&other.negated())
     }
 
+    /// `17.3.20 Temporal.Duration.prototype.round ( roundTo )`
+    ///
+    /// Spec: <https://tc39.es/proposal-temporal/#sec-temporal.duration.prototype.round>
+    ///
+    // Spec last accessed: 2025-05-16, <https://github.com/tc39/proposal-temporal/tree/c150e7135c56afc9114032e93b53ac49f980d254>
```

Comment:
Oooh, this is actually really useful! IMO we should start doing something like this for all our spec-related methods.

(with enough tooling put into it, we could have something that keeps track of the oldest parts of the codebase that could be out of date with the latest specification, but that's just an idea)

Reply:
> (with enough tooling put into it, we could have something that keeps track of the oldest parts of the codebase that could be out of date with the latest specification, but that's just an idea)

I like the idea of a tool for, maybe we should have something more structured so it's easier to parse and harder to mistype, maybe something like:

```
Spec(2025-05-16): <https://github.com/tc39/proposal-temporal/tree/c150e7135c56afc9114032e93b53ac49f980d254>
```

Or maybe just a date so it's easier to type, should be enough info to pin point a commit?

```bash
git log --oneline --since="2025-01-05" --until="2025-01-06"
```